modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc

```diff
@@ -120,82 +120,24 @@ image::ms-dynamics/ms-service-bus-windows-1.png[Connection]
-|Disable Message Timestamp | If true, the message is flagged to avoid generating its sent Timestamp.
-|Delivery Delay | Sets the delivery delay to be applied to postpone message delivery.
-|Delivery Delay Unit | Time unit to be used in the Delivery Delay configurations.
-
-|===
-
-==== Topic Receive
-
-. Click a trigger and select ServiceBus > Topic Receive
-+
-image::ms-dynamics/ms-service-bus-topic-source.png[Queue Source]
-+
-. Fill Global configuration for this connector as we explained in  Operations Section
-. Complete source parameters
-+
-[%header,cols="30s,70a"]
-|===
-|Field |Description
-|Disable Auto Acknowledge | Check this, if you want to Acknowledge events manually with  Acknowledge Message operation
-|Ack Mode | The Session ACK mode to use when consuming a message.
-|Body | The body of the Message.
-|JMS Type | The JMSType identifier header of the Message.
-|Correlation ID | The JMSCorrelationID header of the Message.
-|Send Content Type |Whether or not the body content type should be sent as a property.
-|Content Type | The content type of the Message body.
-|Send Encoding | Whether or not the body outboundEncoding should be sent as a Message property.
-|Encoding | The encoding of the Message body.
-|Reply To - Destination | The destination where a reply to this Message should be sent
-|Reply To - Destination Type | The type of this destination.
-|User Properties | The custom user properties that should be sent to the message
-|JMSX Properties | JMS reserves the 'JMSX' property name prefix for JMS defined properties. Here we define the set of 'well known' properties of JMS.
-|Persistent Delivery | If true, the Message is sent using the PERSISTENT JMSDeliveryMode
-|Priority | The default JMSPriority value to be used when sending the message.
-|Time To Live | Defines the default time the message is in the broker before it expires and is discarded.
-|Time To Live Unit | Time unit to be used in the Time To Live configurations.
-|Disable Message Id | If true, the Message is flagged to avoid generating its MessageID.
-|Disable Message Timestamp | If true, the Message is flagged to avoid generating its sent Timestamp.
-|Delivery Delay | Sets the delivery delay to be applied in order to postpone the Message delivery.
-|Delivery Delay Unit | Time unit to be used in the Delivery Delay configurations.
-|Topic Path | General Topic that you want to listen to a specific subscription.
-|Subscription Path | Subscription where you are going to receive events.
+|Source Type | Indicates whether to retrieve messages from a queue or topic *Is a custom destiniation type also a choice here?*
+|Destination | Name of the destination queue or topic from which messages are retrieved *Is a custom destiniation type also a choice here?*
+|Subscription | Name of the subscription to use when subscribing to topics. If the destination is a queue, select `NONE`.
+|Topic Path |Name of the topic from which messages are retrieved
```

Comment:
Does the Listener have a "Topic Path" parameter...?

Reply:
You are correct - fixed. It doesn't have a Source Path either.